writeup/conStruct.tex

```diff
@@ -453,26 +453,27 @@ \section*{Results}
 \begin{equation} \label{eqn:first_allelic_cov}
     \widehat{\Omega}_{i,j} = \cov[X_i, X_j] .
 \end{equation}
+Each $X_i$ behaves marginally as a fair coin --
+in particular, $\P\{X_i = 1\} = 1$, so $\widehat{\Omega}_{i,i} = 1/4$ for every $i$ --
+all information enters through \emph{correlations}.
+
 
 Although we describe this as a covariance between individually drawn alleles,
 $\widehat{\Omega}_{i,j}$ is in fact also the covariance between the allele frequencies
-of a randomly chosen allele in samples $i$ and $j$, as long as $i \neq j$
-(we discuss the diagonal of $\widehat{\Omega}_{i,j}$
-further in \secref{allelic_cov}).
-
+of a randomly chosen allele in samples $i$ and $j$, as long as $i \neq j$.
 The choice of allele does not affect subsequent calculations, and so may be arbitrary.
-The allelic covariance, $\widehat \Omega$, can be calculated as a covariance
+$\widehat \Omega$, can be calculated as a covariance
```

Comment:
we've said "allelic covariance, Omega" enough times that I figured just Omega would do it.